### ISSUE TITLE
Weird numbers on top-left corner

### ISSUE THREAD

**Issue by hlepnamingblyat** [Monika-After-Story/MonikaModDev#9807]
What are they?

**Comment by Wingdinggaster656** [comment]
At least give a screenshot, or let us continue to guess into the void.

**Comment by dreamscached** [comment]
RenPy labels seen/labels total.

**Comment by Booplicate** [comment]
Press `F2` to hide it.

**Comment by hlepnamingblyat** [comment]
> Press `F2` to hide it.


It worked. Is it like what dreamscached said? Being labels? What does that mean? (srry, I barely know anything about RenPy)

**Comment by Booplicate** [comment]
Yes, just debug information, perhaps we should remove it as it's meaningless in MAS.
